saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -308,3 +312,245 @@ def perform_mutation(cls, _root, info, **data):
         fulfillment.refresh_from_db(fields=["status"])
         order.refresh_from_db(fields=["status"])
         return FulfillmentCancel(fulfillment=fulfillment, order=order)
+
+
+class OrderRefundLineInput(graphene.InputObjectType):
+    order_line_id = graphene.ID(
+        description="The ID of the order line to refund.", name="orderLineId"
+    )
+    quantity = graphene.Int(
+        description="The number of items to be refunded.", required=True,
+    )
+
+
+class OrderRefundFulfillmentLineInput(graphene.InputObjectType):
+    fulfillment_line_id = graphene.ID(
+        description="The ID of the fulfillment line to refund.",
+        name="fulfillmentLineId",
+    )
+    quantity = graphene.Int(
+        description="The number of items to be refunded.", required=True,
+    )
+
+
+class OrderRefundProductsInput(graphene.InputObjectType):
+    order_lines = graphene.List(
+        graphene.NonNull(OrderRefundLineInput),
+        description="List of unfulfilled lines to refund.",
+    )
+    fulfillment_lines = graphene.List(
+        graphene.NonNull(OrderRefundFulfillmentLineInput),
+        description="List of fulfilled lines to refund.",
+    )
+    amount_to_refund = PositiveDecimal(
+        required=False,
+        description=(
+            "The total amount of refund when the value is provided manually. The "
+            "amountToRefund and includeShippingCosts are mutually exclusive and "
+            "attempts to call mutation with both options will raise an exception."
+        ),
+    )
+    include_shipping_costs = graphene.Boolean(
+        description=(
+            "If true, Saleor will refund shipping costs. The amountToRefund and "
+            "includeShippingCosts are mutually exclusive and attempts to call mutation "
+            "with both options will raise an exception."
+        ),
+        default_value=False,
+    )
+
+
+class FulfillmentRefundProducts(BaseMutation):
+    fulfillment = graphene.Field(Fulfillment, description="A refunded fulfillment.")
+    order = graphene.Field(Order, description="Order which fulfillment was refunded.")
+
+    class Arguments:
+        order_id = graphene.ID(
```

Comment:
In Arguments we are using just `id` instead of `order_id`.

Reply:
The `id` would be fine if we would use `fulfillment` object here, not `order`. I was using the `OrderFulfill` mutation as an example.

Reply:
But you are right with the suffix.